app/javascript/angular/code/controllers/_sessions_list_ctrl.js

```diff
@@ -43,16 +42,10 @@ export const SessionsListCtrl = (
     "params.get('map')",
     ({ hasChangedProgrammatically }) => {
       console.log("watch - params.get('map')");
-      if (sessionsUtils.isSessionSelected() && !firstLoad) return;
+      if (sessionsUtils.isSessionSelected()) return;
       // when loading the page for the first time sometimes the watch is triggered twice, first time with hasChangedProgrammatically as undefined
       if (hasChangedProgrammatically === undefined) return;
 
-      if (firstLoad) {
-        sessions.fetch({ amount: params.paramsData["fetchedSessionsCount"] });
```

Comment:
Inside `fetch` there's a call to `map.getBounds()` that returns the wrong object if GoogleMaps didn't have time to initialize. Up until now the debounce was adding 750 ms of delay that gave time to to GoogleMaps to initialize